imports/plugins/included/payments-wallet/client/containers/exampleSettingsFormContainer.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Meteor } from "meteor/meteor";
+import { Packages } from "/lib/collections";
+import { TranslationProvider } from "/imports/plugins/core/ui/client/providers";
+import { Reaction, i18next } from "/client/api";
+import { ExampleSettingsForm } from "../components";
+
+class ExampleSettingsFormContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      apiKey: "278302390293"
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.saveUpdate = this.saveUpdate.bind(this);
+  }
+
+  handleChange(e) {
```

Comment:
Use arrow function for the class methods so as not to specifically bind in the constructor. The constructor will then be useless in this case, therefore it should be deleted. E.g.
```js
state = {
  apiKey: "278302390293"
};

handleChange = (e) => {...}
```
Do the same for `handleSubmit` and `saveUpdate`